### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by j8xixo12**
Add turgon requirements, build process and test into Readme.

### CODE REVIEW

**Comment by yungyuc** [comment]
Thanks a lot for the contribution.  It seemed that you created a new readme file (in markdown format) and deleted the existing readme file (in [rst format])(https://docutils.sourceforge.io/rst.html); https://github.com/yungyuc/turgon/blob/master/README.rst).  Could you update your PR to use the existing README.rst file and the rst format?

**Comment by j8xixo12** [comment]
> [#20 (comment)](https://github.com/yungyuc/turgon/pull/20#issuecomment-663913679) should have come here, so repost for clarity: Let's use the existing rst format README instead of the new md file.

Got it!!
I'll rewrite the doc in rst format.

File: README.rst [hunk 1]
```diff
@@ -3,4 +3,30 @@ The conservation element and solution element (CESE) method
 ===========================================================
 
 .. image:: https://mybinder.org/badge_logo.svg
-  :target: https://mybinder.org/v2/gh/yungyuc/turgon/master?filepath=notebook
\ No newline at end of file
+  :target: https://mybinder.org/v2/gh/yungyuc/turgon/master?filepath=notebook
+
+Build Turgon on OS X
+====================
+Clone from https://github.com/yungyuc/turgon.git:
```
Comment (yungyuc) [inline README.rst]: There should be a blank line between the above section mark '=====' and this line

File: README.rst [hunk 2]
```diff
@@ -3,4 +3,30 @@ The conservation element and solution element (CESE) method
...
+ `$ git clone https://github.com/yungyuc/turgon.git`
+
+Turgon needs following packages:
```
Comment (yungyuc) [inline README.rst]: You are making a 'definition list' in this section.  It doesn't look like a good way to format the sequence of actions.  Could you change to use simple paragraphs?

Note that there should be a blank line between `.. code-block::` and it's previous line.

File: README.rst [hunk 3]
```diff
@@ -3,4 +3,30 @@ The conservation element and solution element (CESE) method
...
 
 .. image:: https://mybinder.org/badge_logo.svg
-  :target: https://mybinder.org/v2/gh/yungyuc/turgon/master?filepath=notebook
\ No newline at end of file
+  :target: https://mybinder.org/v2/gh/yungyuc/turgon/master?filepath=notebook
+
+Build Turgon on OS X
+====================
+Clone from https://github.com/yungyuc/turgon.git:
+ `$ git clone https://github.com/yungyuc/turgon.git`
```
Comment (yungyuc) [inline README.rst]: For code, please use `code-block`: https://www.sphinx-doc.org/en/1.8/usage/restructuredtext/directives.html#directive-code-block.  There are several places to update.

File: README.rst [hunk 4]
```diff
@@ -3,4 +3,30 @@ The conservation element and solution element (CESE) method
...
+
+Now change the current directory to `turgon/spacetime`:
+ `$ cd turgon/spacetime`
+
+Run make:
+ `$make CMAKE_ARGS=-Dmodmesh_INCLUDE_DIR=path_to/modmesh/include`
+
+Test
+====
+Using gtest to test if turgon is installed properly:
```
Comment (yungyuc) [inline README.rst]: ditto

Comment (j8xixo12) [reply README.rst]: > Please also avoid the change merging from `master` branch. Instead, you can rebase and force push to your branch.
> 
> And it seems that you are working in your own `master` branch. Working in your own `master` branch can easily cause confusion after you create multiple PRs. I'd suggest to use a feature branch (which could use a name like `feature/update-readme`) next time.

Sorry, I'll drop the habit.

File: README.rst [hunk 5]
```diff
@@ -3,4 +3,45 @@ The conservation element and solution element (CESE) method
...
+
+.. code-block:: bash
+
+  $ cd turgon/spacetime
+
+Run make:
+
+.. code-block:: bash
+
+   $ make CMAKE_ARGS=-Dmodmesh_INCLUDE_DIR=path_to/modmesh/include
```
Comment (yungyuc) [inline README.rst:38]: Please use the same indentation (2 chars) as that at line 15 and 32.

File: README.rst [hunk 6]
```diff
@@ -3,4 +3,45 @@ The conservation element and solution element (CESE) method
...
+
+Test
+====
+
+Using gtest to test if turgon is installed properly.
+
+.. code-block:: bash
+
+   $ make test
```
Comment (yungyuc) [inline README.rst:47]: ditto

**Pull request merged**